src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -56,12 +57,105 @@ void createStation() {
      * Then 2개의 지하철역을 응답 받는다
      */
     // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("2개의 지하철역을 생성한 다음, 2개의 지하철역을 조회한다.")
+    @Test
+    void showStation() {
+        // given
+        List <String> createdStationNameList = new ArrayList<>(List.of("강남역", "역삼역"));
+
+        for(int i = 0; i < createdStationNameList.size(); i ++) {
+
+            Map<String, String> createdStationParams = new HashMap<>();
+            createdStationParams.put("name", createdStationNameList.get(i));
+
+            ExtractableResponse<Response> createdStationResponse =
+                    RestAssured.given().log().all()
+                            .body(createdStationParams)
+                            .contentType(MediaType.APPLICATION_JSON_VALUE)
+                            .when().post("/stations")
+                            .then().log().all()
+                            .extract();
```

Comment:
해당 코드를 매번 for로 반복하면서 테스트 내에 작성하기보다는 함수로 분리해보는건 어떨까요? 

```suggestion
        ExtractableResponse<Response> response = 지하철_노선_등록("강남역");
        응답_결과_확인(response, HttpStatus.CREATED);
```

이런식으로 말이죠 ㅎ